secrets-provider-test-chart/templates/app.yaml

```diff
@@ -0,0 +1,29 @@
+# Not to be shipped. Created for end-to-end testing
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: test-env
+  name: test-env
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: test-env
+  template:
+    metadata:
+      labels:
+        app: test-env
+    spec:
+      serviceAccountName: cucumber
+      containers:
+      - image: centos:7
+        name: test-app
+        command: ["sleep"]
+        args: ["infinity"]
+        env:
+          - name: TEST_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: test-k8s-secret
```

Comment:
please, remove "test" prefix everywhere

Reply:
this is a dummy app created for testing purposes. I am removing this (: